### DISCUSSION TITLE
Updated To Latest Version - One Device Went Offline - Can't ReAdd

### DISCUSSION THREAD

**Discussion by paterson37** [petretiandrea/home-assistant-tapo-p100#462] [Q&A]
Hi,

So I updated to the latest version of the integration. I noticed that one of my Smart Plug went offline and I couldn't get it back working. So I removed it and tried to re-add it. However I'm not getting an invalid credentials error. All my details are still the same and I'm getting the ip address of both the app and my router.

V2.4.0
Email contains no capital letters.
No 2FA.

Does anyone know what may be going on here?

Thanks folks!

**Comment by PaulTheBlur** [comment]
Same here but I lose about 50% of my devices every upgrade.  Very annoying.

**Comment by WelshWizzard** [comment]
Same here thought it was an ip address change on one of the devices, but looks like it’s another issue.
